### DISCUSSION TITLE
请问srt字幕怎样改回原始样式？

### DISCUSSION THREAD

**Discussion by tenkirain** [diana7127/mpv.net-DW#20] [Q&A]
请问srt字幕怎样改回原始样式？

**Comment by diana7127** [comment]
删掉mpv.conf文件里的`profile = SRT-Drama`

**Reply by philip-fhjkladsh** [reply]
我想再问一下怎么改srt字幕显示时的边框啊。我更喜欢窄一点的，想改一下。

**Reply by diana7127** [reply]
mpv.conf文件里添加`sub-border-size  = 1`，数字可改，默认为3
